packages/tale_knowledge/tests/test_splitter_reassembly.py

```
@@ -0,0 +1,218 @@
+
+# ---------------------------------------------------------------------------
+# Determinism + edge cases
+# ---------------------------------------------------------------------------
+
+
+def test_reindex_idempotency():
+    """Running chunk_content twice on the same input produces identical output."""
+    a = chunk_content(PROSE, chunk_size=300, chunk_overlap=40)
+    b = chunk_content(PROSE, chunk_size=300, chunk_overlap=40)
+    assert len(a) == len(b)
+    for x, y in zip(a, b):
```

Ruff (0.15.11)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)